### DISCUSSION TITLE
Stop showing a 404 on the root of the custom domain

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#649] [Anything Else]
Custom domain root when ready to use shows a 404 error. Two people emailed asking about this today so adding it here too. I've updated the docs to say people should not worry about it but perhaps it's something to stop displaying when we do the upgrade to the custom domain infrastructure. It could redirect to some other place such as our home page instead. 

![Screenshot from 2021-01-25 17-45-54](https://user-images.githubusercontent.com/34340819/105736793-770a4800-5f35-11eb-94af-48101b922fb0.png)

**Comment by Vigasaurus** [comment]
I know a lot of people have requested it to redirect to the actual stats page for the site, I think it would be neat for it to redirect there if the site is public, or to https://plausible.io/sites if not.

I do of course recognize that querying the Postgres instance to check this from the custom domain box may be difficult, so maybe the latter all the time would be beneficial instead.

**Reply by metmarkosaric** [reply]
yeah redirecting to the public dashboard or to /sites would work too!
